### ISSUE TITLE
Error when reviewing rescheduled new cards with sidebar opened

### ISSUE THREAD

**Issue by ryryjy** [noobj2/Anki-Advanced-Review-Bottombar#36]
I've noticed this happening whenever I reschedule a new card from the android app then syncing it back to the PC app to review later. Then I tried rescheduling a new card on desktop, and it showed the same error. However, this doesn't happen without the sidebar. 

```
Anki 2.1.49 (dc80804a) Python 3.8.6 Qt 5.14.2 PyQt 5.14.2
Platform: Windows 10
Flags: frz=True ao=True sv=2
Add-ons, last update check: 2023-04-15 18:44:59

Caught exception: //old ver.
Traceback (most recent call last):
  File "aqt\webview.py", line 41, in cmd
  File "aqt\webview.py", line 142, in _onCmd
  File "aqt\webview.py", line 595, in _onBridgeCmd
  File "aqt\overview.py", line 91, in _linkHandler
  File "aqt\main.py", line 672, in moveToState
  File "aqt\main.py", line 694, in _reviewState
  File "aqt\reviewer.py", line 141, in show
  File "aqt\reviewer.py", line 161, in refresh_if_needed
  File "aqt\reviewer.py", line 211, in nextCard
  File "aqt\reviewer.py", line 344, in _showQuestion
  File "aqt\hooks_gen.py", line 3142, in __call__
  File "anki\hooks.py", line 34, in runHook
  File "C:\Users\***\AppData\Roaming\Anki2\addons21\1136455830\Card_Info.py", line 61, in show
    self._update()
  File "C:\Users\***\AppData\Roaming\Anki2\addons21\1136455830\Card_Info.py", line 384, in _update
    txt += d.cardStats_mod(current_card)
  File "C:\Users\***\AppData\Roaming\Anki2\addons21\1136455830\Card_Info.py", line 320, in cardStats_mod
    return CardStats(self, card).report_mod()
  File "C:\Users\***\AppData\Roaming\Anki2\addons21\1136455830\Card_Info.py", line 265, in report_mod
    pressed_all = pressed_again + pressed_hard + pressed_good + pressed_easy
TypeError: unsupported operand type(s) for +: 'NoneType' and 'NoneType'



Caught exception: // latest ver.
Traceback (most recent call last):
  File "aqt\main.py", line 207, in on_focus_changed
  File "aqt\hooks_gen.py", line 2567, in __call__
  File "aqt\main.py", line 749, in on_focus_did_change
  File "aqt\reviewer.py", line 161, in refresh_if_needed
  File "aqt\reviewer.py", line 211, in nextCard
  File "aqt\reviewer.py", line 344, in _showQuestion
  File "aqt\hooks_gen.py", line 3142, in __call__
  File "anki\hooks.py", line 34, in runHook
  File "C:\Users\***\AppData\Roaming\Anki2\addons21\1136455830\Card_Info.py", line 61, in show
    self._update()
  File "C:\Users\***\AppData\Roaming\Anki2\addons21\1136455830\Card_Info.py", line 384, in _update
    txt += d.cardStats_mod(current_card)
  File "C:\Users\***\AppData\Roaming\Anki2\addons21\1136455830\Card_Info.py", line 320, in cardStats_mod
    return CardStats(self, card).report_mod()
  File "C:\Users\***\AppData\Roaming\Anki2\addons21\1136455830\Card_Info.py", line 266, in report_mod
    self.addLine("Again", "{} | {:.0f}%".format(str(pressed_again).rjust(4), float(pressed_again/pressed_all)*100))
ZeroDivisionError: division by zero
```

**Comment by ryryjy** [comment]
Hello! Thank you for the early reply. The steps are simple.
1. Create a new card and input anything in its fields.
2. Reschedule that card to 0 in the Browse window.
3. Study on that card then open the sidebar.
I've also attached a recording for a clear demonstration

https://user-images.githubusercontent.com/36298836/153740960-efafc420-063c-460a-bc54-081363b559f2.mp4

**Comment by noobj2** [comment]
It should be fixed now

**Comment by ryryjy** [comment]
Hello again! Thank you for fixing such a minor problem of mine. The PC app rescheduling part was fixed. However, new cards rescheduled from AnkiDroid then synced back to the PC app still gives the same error. 

Here are the steps I took:
1. Make a new card in AnkiDroid
2. Reschedule that new card to 0 within AnkiDroid
3. Sync from AnkiDroid, then sync from PC
4. Review that rescheduled card in PC app

I couldn't test this on iOS, sadly, for not having any. 
No rush on this one, and have a good day
